### DISCUSSION TITLE
PSQL: "error: could not translate host name"

### DISCUSSION THREAD

**Discussion by websmyths** [supabase/supabase#31138] [Questions]
I am trying to connect to a newly created database. I tried the psql connection string I got from clicking the "Connect" button in my project. Here's what I got: 

```
psql -h db.wqkhyljzmjytmmtxukts.supabase.co -p 5432 -d postgres -U postgres
psql: error: could not translate host name "db.myhost.supabase.co" to address: nodename nor servname provided, or not known
```
I was unable to ping it: 

```
~/.ssh 1 block
Check my DNS settings
Let's check your local DNS settings by examining the nameserver configuration in /etc/resolv.conf and also test with alternative DNS servers to ensure it's not a local DNS resolution issue.
```

I was unable to dig the host using Google's server, so it is not a local connection problem:

```
dig @8.8.8.8 db.myhost.supabase.co

; <<>> DiG 9.10.6 <<>> @8.8.8.8 db.myhost.supabase.co
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 31850
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;db.myhost.supabase.co. IN	A

;; AUTHORITY SECTION:
supabase.co.		30	IN	SOA	christina.ns.cloudflare.com. dns.cloudflare.com. 2359662825 10000 2400 604800 30

;; Query time: 66 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Sat Dec 14 05:52:06 CST 2024
;; MSG SIZE  rcvd: 131
```

I have two databases with the same problem. Created the new one to see if it was a configuration error in the first one.

**Comment by websmyths** [comment]
Was using this as a troubleshooting step for a database I couldn't get running on Huggingface. Resolved the issues.

**Reply by jhfnetboy** [reply]
So how did you resolve this question? I got the same question: psql: error: could not translate host name "db.ihxwzyizrwluxbvrfcyj.supabase.co" to address: nodename nor servname provided, or not known

**Reply by Abhishek21k** [reply]
https://supabase.com/docs/guides/database/psql
